endstate_correction/analysis.py

```diff
@@ -87,13 +89,55 @@ def plot_results_for_equilibrium_free_energy(
     plt.close()
 
 
+def return_neq_correction(
+    results: Results, method: str = "NEQ", direction: str = "forw"
+) -> Tuple[float, float]:
+    # generate docstring
+    """Return the endstate correction for a given method and direction.
+    Args:
+        results (Results): instance of the Results class
+        method (str): NEQ or MBAR
+        direction (str): forw or rev
+    Returns:
+        float: endstate correction delta_f
+        float: endstate correction error
+    """
+    assert type(results) == Results
+    assert method in ["NEQ", "FEP"]
+    print(f"method: {method}, direction: {direction}")
+    if method == "FEP" and direction == "forw":
+        print(
+            f"Zwanzig's equation (from mm to qml): {exp(results.dE_reference_to_target)['Delta_f']}"
+        )
+        est = exp(results.dE_reference_to_target)
+        return est["Delta_f"], est["dDelta_f"]
+    if method == "FEP" and direction == "rev":
```

Comment:
Might be good to have elif here and a final else which raises exception

Reply:
good point! added the `else` clause.